doc/classes/ItemList.xml

```diff
@@ -57,6 +57,13 @@
 				Ensure current selection is visible, adjusting the scroll position as necessary.
 			</description>
 		</method>
+		<method name="force_update_list_size">
+			<return type="void" />
+			<description>
+				Forces an update to the list size based on its items. If [member auto_height] is [code]true[/code], [method Control.get_minimum_size]'s y value returns the auto height value if [member Control.custom_minimum_size] is less than the auto height.
+				Scrollbar is adjusted accordingly if necessary.
```

Comment:
```suggestion
				Forces an update to the list size based on its items. This happens automatically whenever size of the items, or other relevant settings like [member auto_height], change. The method can be used to trigger the update ahead of next drawing pass.
```

I feel like the rest of the added description isn't necessary here. It just tells that the method acts like it's supposed to act, which is to be expected.